app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
kindly ensure that the application checks for the confirmation of the email. You can do this by adding `:confirmable` to the code on lines 4 and 5.

Reply:
@jerryowusu 
thanks for your good and valuable feedback,
now the application will check for confirmation.